Cargo.toml

```diff
@@ -61,33 +61,34 @@ rand = "0.8.5"
 rand_chacha = "0.3.1"
 rand_mt = "4.2.2"
 regex_generate = "0.2.3"
+usc = { git = "https://github.com/software-mansion/universal-sierra-compiler", rev = "581402f", package = "universal-sierra-compiler" }
```

Comment:
It's been released as a crate: https://spaceshard.slack.com/archives/C03031Y0LKC/p1711116989370019

Reply:
[67fc2be](https://github.com/0xSpaceShard/starknet-devnet-rs/pull/391/commits/67fc2be0d9acbabd62c26789f9f6a4fb0703f4bd)